### PR TITLE
Use Centralized Test Configuration to Define Test Matrix

### PR DESCRIPTION
**Pull request by kythant**
This will help us streamlines the process of updating our TestMatrix across all branches

### CODE REVIEW

File: build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml [hunk 1]
```diff
@@ -132,14 +132,7 @@ stages:
     parameters:
       jobName: PipelineTestsX64
```
Comment (alexlamtest) [inline build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml:133]: jobName: PipelineTests?

**Review by alexlamtest — APPROVED** [review]
LGTM

File: build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml (hunk 1 above)
Comment (kythant) [reply build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml:133]: Oh correct! I need to rename these

File: build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml [hunk 2]
```diff
@@ -132,14 +132,7 @@ stages:
...
       isArm64Platfrom: false
```
[the remark this answers is not in the document]
Comment (kythant) [reply build/AzurePipelinesTemplates/WindowsAppSDK-BuildTransportPackage-Stage.yml:134]: Don't think so!

**Pull request merged**